src/client/snippets/paportal-liquid.code-snippets

```diff
@@ -87,5 +87,75 @@
 		],
 		"description": "Fully renders a web form by name"
-	}
+	},
+    "1 column section": {
+        "prefix": "1-column-section",
+        "body": [
+            "<div class=\"row sectionBlockLayout\" style=\"text-align:left;min-height:374px;padding:8px;\"><div class=\"container\" style=\"display:flex;flex-wrap:wrap;\"><div class=\"col-md-12 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div></div></div>"
+        ],
+        "description": "Fully render a section component with 1 column"
+    },
+    "2 column section": {
+        "prefix": "2-column-section",
+        "body": [
+            "<div class=\"row sectionBlockLayout\" style=\"text-align:left;min-height:374px;padding:8px;\"><div class=\"container\" style=\"display:flex;flex-wrap:wrap;\"><div class=\"col-md-6 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div><div class=\"col-md-6 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div></div></div>"
+        ],
+        "description": "Fully render a section component with 2 column"
+    },
+    "3 column section": {
+        "prefix": "3-column-section",
+        "body": [
+            "<div class=\"row sectionBlockLayout\" style=\"text-align:left;min-height:374px;padding:8px;\"><div class=\"container\" style=\"display:flex;flex-wrap:wrap;\"><div class=\"col-md-4 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div><div class=\"col-md-4 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div><div class=\"col-md-4 columnBlockLayout\" style=\"padding:16px;margin: 60px 0;min-height:200px;\"></div></div></div>"
+        ],
+        "description": "Fully render a section component with 3 column"
+    },
+    "1/3 left column section": {
+        "prefix": "one-third-left-column-section",
+        "body": [
+            "<div class=\"row sectionBlockLayout\" style=\"text-align:left;min-height:374px;\"><div class=\"container\" style=\"padding:0px; display:flex;flex-wrap:wrap;\"><div class=\"col-md-4 columnBlockLayout\" style=\"padding:16px; min-height:200px;\"></div><div class=\"col-md-8 columnBlockLayout\" style=\"padding:16px; min-height:200px;\"></div></div></div>"
+        ],
+        "description": "Fully render a section component with one third left column"
+    },
+    "1/3 right column section": {
+        "prefix": "one-third-right-column-section",
+        "body": [
+            "<div class=\"row sectionBlockLayout\" style=\"text-align:left;min-height:100px;\"><div class=\"container\" style=\"padding:0px; display:flex;flex-wrap:wrap;\"><div class=\"col-md-8 columnBlockLayout\" style=\"padding:16px; min-height:200px;\"></div><div class=\"col-md-4 columnBlockLayout\" style=\"padding:16px; min-height:200px;\"></div></div></div>"
+        ],
+        "description": "Fully render a section component with one third right column"
+    },
+    "image component": {
```

Comment:
This key is displayed in vscode right? If yes it should this be "Image Component"

Reply:
Refer the other key's declared in the file above

Reply:
Yes, I will capitalise the names.